lib/osf-components/addon/components/contributor-list/component.ts

```diff
@@ -51,4 +53,9 @@ export default class ContributorList extends Component {
     get rest(this: ContributorList): number {
         return this.contributors.meta.total - this.max;
     }
+
+    @computed('truncated')
+    get truncate(this: ContributorList): number | undefined {
```

Comment:
You don't need to type `this` and the return type is inferred (it's actually `3 | undefined`).
```suggestion
    get truncate() {
```